extensions/tags/js/src/forum/components/TagsPage.js

```diff
@@ -8,6 +8,7 @@ import humanTime from 'flarum/common/helpers/humanTime';
 import tagIcon from '../../common/helpers/tagIcon';
 import tagLabel from '../../common/helpers/tagLabel';
 import sortTags from '../../common/utils/sortTags';
+import classList from '@flarum/core/src/common/utils/classList';
```

Comment:
for some reason even my IDE started auto importing using `@flarum/core` 🤔 
```suggestion
import classList from 'flarum/common/utils/classList';
```

Reply:
Hmm... Strange...